src/scripting.rs

```diff
@@ -24,10 +24,13 @@ impl Script {
     }
 }
 
-// Quote all strings expect for explicit bash variable references
+// Quote all strings expect for explicit bash variable references and
+// redirection.
 fn quoted(src: &String) -> String {
     if src.starts_with("${") {
         format!(r#""{src}""#)
+    } else if src.starts_with('>') {
+        src.to_string()
```

Comment:
Without this the script comes out with something that looks like:

```bash
...
'>' \
<output_path>
```

The quotation breaks the redirect. This fixes it, but maybe it's not the best way to do it.

Reply:
Also, the extra `else if` here doesn't look too pretty.

Its possible to do with a `match` as well, but I'm not entirely convinced it looks any better. I'm more leaning towards leaving it as-is or doing differently entirely.

```rust
fn quoted(src: &String) -> String {
    match src {
        _ if src.starts_with("${") => format!(r#""{src}""#),
        _ if src.starts_with('>') => src.to_string(),
        _ => yash_quote::quoted(src).to_string(),
    }
}

```

Reply:
this is the perfect moment to introduce tests for this little function